### PR TITLE
feature(emoji): help section

### PR DESCRIPTION
**Pull request by PAKovalev**
Help section now contains Emoji block.
AT-26 [https://momentum.nifty.pm/l/vLDM!oLnPO](https://momentum.nifty.pm/l/vLDM!oLnPO)

### CODE REVIEW

File: src/scenes/widgets/pages/HelpWidget/components/Emoji/Emoji.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          </styled.HighlightedSpan>
+          {t('helpSection.emoji.paragraphs.two.partTwo')}
+        </styled.TextItem>
+        <styled.EmojiButtonContainer>
+          <styled.SmileContainer>
+            <IconSvg name="smiley-face" size="large" />
+          </styled.SmileContainer>
+          <styled.ArrowContainer>
+            <IconSvg name="arrowLeftBig" size="large" />
+            <Heading label="Click here" type="h3" align="left" transform="uppercase" />
```
Comment (aminbksi) [inline src/scenes/widgets/pages/HelpWidget/components/Emoji/Emoji.tsx:62]: translation

File: src/scenes/widgets/stores/HelpStore/HelpStore.ts [hunk 2]
```diff
@@ -1,35 +1,56 @@
-import {types} from 'mobx-state-tree';
+import {cast, flow, types} from 'mobx-state-tree';
 
-import {DialogModel, ResetModel} from 'core/models';
+import {appVariables} from 'api/constants';
+import {api, EmojiConfigResponse} from 'api';
+import {DialogModel, RequestModel, ResetModel} from 'core/models';
 
 export enum HelpSectionType {
```
Comment (aminbksi) [inline src/scenes/widgets/stores/HelpStore/HelpStore.ts:7]: move to enums

**Review by aminbksi — APPROVED** [review]

File: src/scenes/widgets/stores/HelpStore/HelpStore.ts (hunk 2 above)
Comment (PAKovalev) [reply src/scenes/widgets/stores/HelpStore/HelpStore.ts:7]: Very special enum. No need.

File: src/scenes/widgets/pages/HelpWidget/HelpWidget.tsx [hunk 3]
```diff
@@ -8,19 +8,24 @@ import momentum from 'static/images/momentum.svg';
...
   const theme = useTheme();
 
   useEffect(() => {
-    return helpStore.resetModel;
-  }, [helpStore]);
+    helpStore.init(worldStore.worldId);
+
+    return () => {
+      helpStore.resetModel();
+    };
```
[the remark this answers is not in the document]
Comment (PAKovalev) [reply src/scenes/widgets/pages/HelpWidget/HelpWidget.tsx:27]: More clear.

**Pull request merged**